### PR TITLE
Fix Microsoft.Extensions prebuilts in runtime

### PR DESCRIPTION
**Pull request by crummel**

### CODE REVIEW

File: patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Subject: [PATCH 26/26] Patch MS.Extensions.Logging version.
+
+---
+ .../wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj    | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj b/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
+index ea99c465cbb..1658255f787 100644
+--- a/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
++++ b/src/mono/wasm/debugger/BrowserDebugProxy/BrowserDebugProxy.csproj
```
Comment (dagood) [inline patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch:13]: Is it surprising that this mono project is being restored at all?

**Review by dagood — APPROVED** [review]

File: patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@@ -5,7 +5,7 @@
+   </PropertyGroup>
+ 
+   <ItemGroup>
+-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.2" />
++    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingPackageVersion)" />
```
Comment (tmds) [inline patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch:19]: @crummel what is going to set the `MicrosoftExtensionsLoggingPackageVersion` variable?

Comment (crummel) [reply patches/runtime/0026-Patch-MS.Extensions.Logging-version.patch:19]: This should be getting set in SBRP or previously-source-built packages, and Arcade should import then before this gets evaluated.

**Comment by crummel** [comment]
Superseded by 5.0 work.

**Pull request closed without merging**